backend/src/main/java/de/robinschatzl/gameapp/backend/security/UserController.java

```diff
@@ -41,4 +44,17 @@ public void logout(HttpSession httpSession) {
         SecurityContextHolder.clearContext();
     }
 
+    @PostMapping("/signup")
+    //eslint-disable-next-line
```

Comment:
esLint is a javaScript linter => that should make no difference!
You can use @SupressWarnings("Java:93928xyz") but you should only suppress warnings, when you know what you are doing ;)

Reply:
im not sure anymore why i was add this xD